### ISSUE TITLE
Cant craft Fluix-Plated Iron Ingot

### ISSUE THREAD

**Issue by xXDarkNovaXx** [phantamanta44/Lazy-AE2#48]
The advanced Inscriper is eat the Materials for the Fluix-Plated Iron Ingot but is not craft the Ingot
I put 5 materials in the machine but I can't get anything out
https://prnt.sc/zsoocd

The Recipe itself is correct
https://prnt.sc/zspe1c

Sry for Error i use a Translator i come from Germany

**Comment by phantamanta44** [comment]
does it work with the normal inscriber? i'll admit that i've never tried using lazy ae2 and ae2stuff together

**Comment by xXDarkNovaXx** [comment]
The normal inscriber is work hmm
But is Inscriper not Inscriper? 
AE2 Stuff builds on AE2 and the advanced Inscriper seems to recognize the Recipe
I added Lazy AE2 just yesterday

**Comment by phantamanta44** [comment]
yeah, i don't really know why the ae2stuff inscriber wouldn't work. maybe you're accidentally extracting the output through some pipes?

**Comment by xXDarkNovaXx** [comment]
I have no Pipes or Import Bus on the advanced Inscriper
The bar doesn't move either and is say 0%
The advanced inscriber has an animation on the sides from which you can also see when he is working
